### PR TITLE
Improve the README.md

### PR DESCRIPTION
**Pull request by cuzik**
Hi hi :cow: 

In thin PR we improve the README.md file with the instruction to start application and test it.

### CODE REVIEW

**Comment by cuzik** [comment]
[Melhorar o README.md (1)](https://trello.com/c/ZmC6XtfN/35-melhorar-o-readmemd-1)

File: README.md [hunk 1]
```diff
@@ -42,6 +45,32 @@ echo "POSTGRESQL_PASSWORD='YourPassword'" >> .env
...
 bin/rails db:setup
+bin/rails db:migrate
+bin/rails db:seed
+```
+
+## Testing
+
+### Test
+
+We use rspec framework to test our rails applicatio. To test you can run:
```
Comment (rartner) [inline README.md]: Rails application

File: README.md [hunk 2]
```diff
@@ -42,6 +45,32 @@ echo "POSTGRESQL_PASSWORD='YourPassword'" >> .env
...
+
+### Jest
+
+To test our js files we use jest framework. Run with:
+
+```sh
+yarn test
+```
+
+and you change some component you should to update de snapshot with:
```
Comment (rartner) [inline README.md]: this shouldn't be "and if you change"?

Comment (markx3) [reply README.md]: Please change this to: `and if you change any components, you should update the snapshot with:`

PS: `de` 🦑

File: README.md [hunk 3]
```diff
@@ -57,3 +86,7 @@ Run tests with:
 ```sh
 rspec
```
Comment (markx3) [inline README.md]: We are running tests on Travis with bundle exec rake. Considering I'm a noob, does this make a difference?

Comment (markx3) [inline README.md]: We are running tests on Travis with `bundle exec rake`. Does this make any difference? 

(Please, beware of the noob).

File: README.md [hunk 4]
```diff
@@ -42,6 +45,32 @@ echo "POSTGRESQL_PASSWORD='YourPassword'" >> .env
...
+
+### Jest
+
+To test our js files we use jest framework. Run with:
+
+```sh
+yarn test
+```
+
+and if you change some component you should to update de snapshot with:
```
Comment (markx3) [inline README.md]: `de`

Comment (markx3) [reply README.md]: `And if you change any components, you should update the snapshot with:`

File: README.md (hunk 3 above)
Comment (cuzik) [reply README.md]: Rake contents the rails tasks. The default task is to run the test suite, that in our case is the `rspec`. When we use `bundle exec rake` we're actually running `bundle exec rspec`. So in that case, whatever command you choose.

Comment (markx3) [reply README.md]: Oh okay =) Thx <3

**Pull request merged**